### PR TITLE
Removal of files option of the cli + dep upgrade + cs

### LINKED ISSUE #174: What is the purpose of "files" sub-command
Currently there is one option. Seems like the idea was to have a `mod` also.
But since we have merged them, can we remove the `files` sub-command also?
Or does it have some other use cases?

### PR DESCRIPTION
**Pull request by omid**
Fixes #174

### CODE REVIEW

**Review by jxs — CHANGES_REQUESTED** [review]
Thanks for this. There are still refferences to the [`files` subcommand](https://github.com/omid/refinery/blob/remove_files_from_cli/refinery_cli/src/migrate.rs#L15-L26) and the cli tests still use it also, that's why they are failing.   
Due to what we discussed on #170 it probably doesn't make sense to support `.rs` migration files here, so we should introduce differentiation between just `.sql` files and both of them similar to what [we had here](https://github.com/rust-db/refinery/blob/cc0ad88469ecdff8f10b2a96e84ea8c5432d04b5/refinery_core/src/util.rs#L10-L24) to be called on just `.sql` files for the `cli` case, sorry I missed it on #170. 
What do you think?

**Comment by omid** [comment]
How come I forgot to touch the logic :D stupid me.

Anyway, about what you said. Sorry, I didn't get it, why should we only run `.sql` files with CLI? Or you mean something else?

**Comment by omid** [comment]
OK, I think I understood what you mean. We should add thee `rs` support to cli. I'm on it.

**Comment by jxs** [comment]
Hi, no worries :grinning: meanwhile I have addressed that issue while working on #177, see https://github.com/jxs/refinery/pull/177/commits/fa28ffc2ec42fb76a76e1beddb882eea5ce8497f and feel free to rebase against `main`

File: refinery_cli/src/migrate.rs [hunk 1]
```diff
@@ -11,31 +11,28 @@ pub fn handle_migration_command(args: &ArgMatches) -> anyhow::Result<()> {
...
 
-fn run_files_migrations(
+fn run_migrations(
     config_location: &str,
     grouped: bool,
     divergent: bool,
     missing: bool,
     env_var_opt: Option<&str>,
-    arg: &ArgMatches,
+    path: &str,
```
Comment (jxs) [inline refinery_cli/src/migrate.rs:34]: thanks for this improvement :ok_hand:

**Review by jxs — APPROVED** [review]
LGTM thanks! :fireworks:

**Comment by omid** [comment]
OK, cool 👍🏼

**Pull request merged**